docs/notebooks/regional_frequency_analysis.ipynb

```diff
@@ -0,0 +1,970 @@
+{
```

Comment:
<ol><li>Il pourrait y avoir un sous-titre ici aussi, puisqu'on change de sujet.</li><li>Typo: <code>exammple</code></li><li>La cellule juste en-dessous calcule des choses pour rien. Plutôt que de faire ça, je modifierais plutôt le Markdown:</li></ol><p> </p><blockquote>Different clustering methods from <code>sklearn.cluster</code> can be used to group the stations together using the results from the PCA. In this example we'll use <code>AgglomerativeClustering</code>, but <code>HDBSCAN</code> or <code>OPTICS</code> would also provide valid results.  The regional clustering itself is performed using <code>xhfa.regional.get_group_from_fit</code>, which can take the arguments of the <code>skleanr</code> functions as a dictionnary.</blockquote><p> </p><p>   Tu pourrais aussi utiliser une cellule pour afficher la doc de la fonction. J'ai commencé à faire ça ailleurs dans mes notebooks aussi:</p>

<pre class="ql-syntax" spellcheck="false">print(xhfa.regional.get_group_from_fit.__doc__)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>